modules/nw-ptp-t-bc-t-tsc-holdover.adoc

```diff
@@ -0,0 +1,369 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/configuring-ptp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ptp-t-bc-t-tsc-holdover_{context}"]
+= Applying unassisted holdover for boundary clocks and time slave clocks
+
+The unassisted holdover feature enables an Intel E810-XXVDA4T Network Interface Card (NIC), configured as either a PTP boundary clock (T-BC) or a PTP time slave clock (T-TSC), to maintain highly accurate time synchronization even when the upstream timing signal is lost. This is achieved by relying on the NIC's internal oscillator to enter a stable, controlled drift state.
+
+The `ts2phc` service monitors the `ptp4l` instance bound to the timing receiver (TR) port. If the TR port stops operating as the time receiver, if upstream grandmaster clock (T-GM) deteriorates in quality or the link disconnects, for example, the system enters holdover mode and reconfigures itself dynamically.
```

Comment:
```suggestion
The `ts2phc` service monitors the `ptp4l` instance bound to the timing receiver (TR) port. If, for example, the TR port stops operating as the time receiver, the upstream grandmaster clock (T-GM) deteriorates in quality or the link disconnects, the system enters holdover mode and reconfigures itself dynamically.
```

I think the three "if's" here are examples so it could be rewritten as: